src/odemis/driver/lakeshore.py

```diff
@@ -507,16 +503,16 @@ def _poll(self):
 
 class SerialBusAccesser(object):
     """
-    Manages connections to the low-level bus
+    Manages serial bus connections
     """
 
     def __init__(self, port: str):
-        # Ensure no one will talk to it simultaneously, and we don't talk to devices already in use
         """
         port: the full path to the serial port (eg: /dev/ttyUSB1)
         raises:
            IOError: if something went wrong opening the port
         """
+        # Ensure no one will talk to it simultaneously, and we don't talk to devices already in use
```

Comment:
That comment is a little bit incorrect or at least prone to misinterpretation. This lock just ensures that this odemis driver will not send multiple command simultaneously. It doest NOT prevent other programs from trying to talk to the device (that's what the file lock does).